apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -143,6 +146,58 @@ expression: "&schema"
       },
       "additionalProperties": false
     },
+    "Exporter": {
+      "oneOf": [
+        {
+          "type": "object",
+          "required": [
+            "grpc"
+          ],
+          "properties": {
+            "grpc": {
+              "$ref": "#/definitions/GrpcExporter",
+              "nullable": true
+            }
+          },
+          "additionalProperties": false
+        }
+      ]
+    },
+    "GrpcExporter": {
+      "type": "object",
+      "required": [
+        "protocol"
+      ],
+      "properties": {
+        "endpoint": {
+          "default": null,
+          "type": "string",
+          "format": "uri",
+          "nullable": true
+        },
+        "metadata": {
+          "default": null,
+          "type": "object",
+          "additionalProperties": true,
+          "nullable": true
+        },
+        "protocol": {
+          "$ref": "#/definitions/ProtocolMirror",
+          "nullable": true
+        },
+        "timeout": {
+          "type": "integer",
+          "format": "uint64",
+          "minimum": 0.0,
+          "nullable": true
+        },
+        "tls_config": {
+          "$ref": "#/definitions/TlsConfig",
+          "nullable": true
+        }
+      },
+      "additionalProperties": false
+    },
```

Comment:
What are all these changes to the snapshots about Grpc and Otel and stuff?

Reply:
a676ff45 got the root cause and fixed the test!